docs/tuning_fledge.rst

```diff
@@ -30,6 +32,42 @@ Many factors will impact the performance of a Fledge system
 
 Many of these are outside of the control of Fledge itself, however it is possible to tune the way Fledge will use certain resources to achieve better performance within the constraints of a deployment environment.
 
+Setting Log Level
+=================
+
+Fledge writes logs via the *syslog* facility of Linux, this allows for multiple different log levels. Altering the log level will impact the performance of the system and can use significant disk space when set to *debug* or *info* levels. Each of the services within a Fledge instance may have the log level set of that service individually.
+
+The logging level for the Fledge core can be set in the *Logging* configuration category in the *Advanced* parent category that can be accessed from the *Configuration* menu item.
+
++------------------+
+| |core_log_level| |
++------------------+
+
+The logging level can be set to *error*, *warning*, *info* or *debug*, the default setting is *warning*. The level set defines the least severe error that will be logged, logs or higher severity that that set will also be logged. In the default setting both *error* and *warning* level logs will be sent to the syslog facility.
```

Comment:
In core - we have one more logging level support i.e 'critical'

At the moment we haven't used this level in python core.

Consistency wise on both sides, We may need to remove this option level from core logging?

Reply:
We should probably drop the critical option

Reply:
Sure, let me create a JIRA for it